gnovm/pkg/gnolang/precompile.go

```diff
@@ -36,9 +36,11 @@ var stdlibWhitelist = []string{
 	"context",
 	"crypto/md5",
 	"crypto/sha1",
+	"crypto/sha256",
 	"encoding/json",
 	"encoding/base64",
 	"encoding/binary",
+	"encoding/hex",
```

Comment:
Please be aware that while this action will influence the 'gno precompile' operation, it won't immediately make those packages available onchain. 

@harry-hov, would you be able to demonstrate how to adjust the `gno.mod` settings to indicate that the package should not be automatically added to genesis at this time?